bin/oneoff/back_populate_poste_deliveries_hashed_email

```diff
@@ -1,16 +1,33 @@
 #!/usr/bin/env ruby
 
-# This script populates the hashed_email column of the pegasus poste_deliveries table.
+# This script populates the hashed_email column of the pegasus poste_deliveries
+# table.
 
 require File.expand_path('../../../pegasus/src/env', __FILE__)
 require 'digest/md5'
 require src_dir 'database'
 
-def main
-  DB[:poste_deliveries].each do |delivery|
+if ARGV.length != 3
+  puts 'Usage: ./bin/oneoff/back_populate_poste_deliveries_hashed_email '\
+    'start_index end_index batch_size'
+  exit 1
+end
+
+start_index = ARGV[0].to_i
+end_index = ARGV[1].to_i
+batch_size = ARGV[2].to_i
+
+until start_index > end_index
+  puts "Processing from #{start_index} to #{start_index + batch_size}..."
+
+  DB[:poste_deliveries].
+    where(id: start_index..(start_index + batch_size)).
+    each do |delivery|
     hashed_email = Digest::MD5.hexdigest(delivery[:contact_email])
-    DB[:poste_deliveries].where(id: delivery[:id]).update(hashed_email: hashed_email)
+    DB[:poste_deliveries].
+      where(id: delivery[:id]).
+      update(hashed_email: hashed_email)
```

Comment:
Note the inner loop does not inspect `end_index` so this is a bit misleading. If `start_index + batch_size` > `end_index` then records past `end_index` will be updated. 

It is fairly harmless though since the operation is idempotent.